frontend/taipy-gui/src/components/Taipy/AutoLoadingTable.tsx

```diff
@@ -604,6 +605,11 @@ const AutoLoadingTable = (props: TaipyTableProps) => {
                                         <TableCell
                                             key={`head${columns[col].dfid}`}
                                             sortDirection={orderBy === columns[col].dfid && order}
+                                            sx={columns[col].width ? { width: columns[col].width } : {}}
+                                            className={col === "EDIT_COL"
+                                                ? getSuffixedClassNames(className, "-action")
+                                                : getSuffixedClassNames(className, generateHeaderClassName(columns[col].dfid))
+                                            }
                                             sx={columns[col].width ? { width: columns[col].width } : undefined}
```

Comment:
duplicate line

Reply:
Done Sir, Please run the test again. i am looking into the previous test's result and will try to fix it.

Reply:
Where is the className gone ?
Why do you have 2 almost identical sx properties ?